python-sdk/examples/dataset_kitchen_sink.py

```
@@ -0,0 +1,179 @@
+try:
+    import pandas as pd
+except ImportError:
+    print("Warning: pandas not available. Some features will be skipped.")
+    pd = None
+
+import langwatch
+import traceback
+from langwatch.dataset import get_dataset, Dataset, DatasetEntry, GetDatasetOptions
+
+
+def main(messages=None):
```

Ruff (0.14.2)

Unused function argument: `messages`

(ARG001)